master/usage/routereflector/calico-routereflector.md

```diff
@@ -18,7 +18,7 @@ However, it may provide a useful framework for future development.
 These instructions are focused around container-based deployments that use the
 calico/node container image.
 
-For an OpenStack deployment, read [Configuring BIRD as a BGP Route Reflector](bird-rr-config).
+<!---For an OpenStack deployment, read [Configuring BIRD as a BGP Route Reflector](bird-rr-config).-->
```

Comment:
I would prefer to delete this than to comment.  Deletion is neater for the current master doc, and we can still get the content back again from git.